secator/tasks/ffuf.py

```
@@ -103,9 +103,9 @@
 
 		# Check FUZZ keyword
 		data = self.get_opt_value('data') or ''
-		headers = self.get_opt_value('header')
-		if not len(self.inputs) > 1 and 'FUZZ' not in self.inputs[0] and 'FUZZ' not in headers and 'FUZZ' not in data:
-			self.add_result(Warning(message='Keyword FUZZ is not present in the URL, header or body'), print=True, output=True)
+		fuzz_in_headers = any('FUZZ' in v for v in self.headers.values())
+		if not len(self.inputs) > 1 and 'FUZZ' not in self.inputs[0] and not fuzz_in_headers and 'FUZZ' not in data:
```

Pylint (3.3.7)

[convention] Line too long (110/100)

(C0301)